RocketChat/Rocket.Chat.ReactNative
# [NEW] Add Discussions to RoomActionsView

## Proposed changes

Currently there is no way to see all the discussions created on a room. This PR achieve this by adding a `Discussions` action to `RoomActionsView` which shows all the existing Discussions in a room.

## Issue(s)	

Closes #3549

## How to test or reproduce


In order to test this you should have a room (channel or group) with at least one discussion

1. Go to the room 
2. Go to room actions

You should be able to see a Discussions item

## Screenshots

## Types of changes




## Checklist



## Further comments
